core/primitives/src/version.rs

```diff
@@ -164,6 +164,29 @@ pub const PROTOCOL_VERSION: ProtocolVersion = MAIN_NET_PROTOCOL_VERSION;
 #[cfg(feature = "nightly_protocol")]
 pub const PROTOCOL_VERSION: ProtocolVersion = 126;
 
+#[allow(dead_code)]
+const MAIN_NET_PROTOCOL_UPGRADE_VOTING_START: Option<ProtocolUpgradeVotingSchedule> =
```

Comment:
this will also be used for testnet right? If so, shouldn't we just name it `STABLE_PROTOCOL_UPGRADE_VOTING_START`?

Reply:
Let's change everything to `STABLE_PROTOCOL_` then